src/jdk.jdeps/share/classes/com/sun/tools/javap/AttributeWriter.java

```diff
@@ -354,6 +355,16 @@ public Void visitInnerClasses(InnerClasses_attribute attr, Void ignore) {
         return null;
     }
 
+    @Override
+    public Void visitJavaFlags(JavaFlags_attribute attr, Void unused) {
+        println("Extended Flags:");
+        indent(+1);
+        if ((attr.extendedFlags & ACC_REF_DEFAULT) != 0)
+            println("ACC_REF_DEFAULT");
+        indent(-1);
```

Comment:
com.sun.tools.javac.code.Flags#ACC_REF_DEFAULT is not visible here. So com.sun.tools.classfile.AccessFlags has been enhanced to define ACC_REF_DEFAULT - but this is a stop gap